src/Maestro/Maestro.Web/Api/v2018_07_16/Models/BuildData.cs

```diff
@@ -27,13 +29,18 @@ public class BuildData
 
         public Data.Models.Build ToDb()
         {
+            var isGitHubInfo = Repository.IndexOf("github", StringComparison.OrdinalIgnoreCase) >= 0;
+
             return new Data.Models.Build
```

Comment:
This needs to save off the BuildNumber also. Even if the build is from a github repository it will still have an AzureDevOpsBuildNumber.